.github/workflows/js-linting.yml

```diff
@@ -0,0 +1,43 @@
+name: JavaScript Linting
```

Comment:
💅  If we are going to set up a global package.json I will also add a .nvmrc file

Reply:
Added in https://github.com/woocommerce/grow/pull/36/commits/fa4b3463c1bbb7f9196e9c14d8e18b0f1f998f99.